### PR TITLE
vagrant: 2.2.5 -> 2.2.6

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

https://github.com/hashicorp/vagrant/releases/tag/v2.2.6

Generated the `gemset.nix` by converting the `vagrant.gemspec` file to a
different `Gemfile` as described in the previous update PR[1].

[1] https://github.com/NixOS/nixpkgs/pull/64302#issuecomment-516837953

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by redvers** [comment]
@aneeshusa 
@volth

File: pkgs/development/tools/vagrant/gemset.nix [hunk 1]
```diff
@@ -266,58 +314,18 @@
...
-    version = "3.5.0";
-  };
-  rspec-support = {
-    source = {
-      remotes = ["https://rubygems.org"];
-      sha256 = "10vf3k3d472y573mag2kzfsfrf6rv355s13kadnpryk8d36yq5r0";
-      type = "gem";
-    };
-    version = "3.5.0";
-  };
```
Comment (tomfitzhenry) [inline pkgs/development/tools/vagrant/gemset.nix]: Why have these testing dependencies been removed?

Comment (Ma27) [reply pkgs/development/tools/vagrant/gemset.nix]: Didn't add those to the Gemfile I used to generate `gemset.nix`. We don't run the tests anyway, so no reason to keep it, right?

**Comment by Ma27** [comment]
Backported as 1be199f0594503ed8b82d5d8d3ae57ea0c719a93.

**Pull request merged**